### DISCUSSION TITLE
Remove scaled object manifest

### DISCUSSION THREAD

**Discussion by miltlima** [kedacore/keda#5886] [General] [answered]
Hello Everyone, 

When I remove the scaledObject the expect behavior should be the target scale to original replica count, but it isn't happening 

I have been use chart 2.14.2 and test with other version and the same behavior happen 

Thanks

**Comment by zroubalik** [comment, marked as answer]
See `restoreToOriginalReplicaCount` at https://keda.sh/docs/2.14/concepts/scaling-deployments/#advanced

**Reply by miltlima** [reply]
@zroubalik thanks !
